src/main/java/com/hm/achievement/utils/PlayerAdvancedAchievementEvent.java

```diff
@@ -82,7 +83,7 @@ public String getMessage() {
 		return message;
 	}
 
-	public String getCommandMessage() {
+	public List<String> getCommandMessage() {
```

Comment:
Should we still provide the old getter and `PlayerAdvancedAchievementEventBuilder commandMessage(String commandMessage)` Builder method for backwards compatibility? Even though this class is not part of the API package, its usage is suggested in the [Wiki](https://github.com/PyvesB/AdvancedAchievements/wiki/Developers#listening-to-achievement-events), so I think we should be slightly more cautious. The new method could be the following:
```
public List<String> getCommandMessages() {
  return commandMessages;
}
```
And the old one:
```
@Deprecated
public String getCommandMessage() {
  return StringUtils.join(commandMessages, ' ');
}
```
And something similar for the Builder class. What do you think?